webpack.config.js

```diff
@@ -168,9 +168,9 @@ module.exports = (env, argv) => {
               options: {
                 modules: {
                   auto: true, // only for files containing ".module." in name
-                  // class name to hash, but also keep name in development
+                  // class name to hash, but also keep name for debug in dev
                   localIdentName: `${
-                    isDev ? '[local]#' : ''
+                    isDev ? '[local]🐛' : ''
```

Comment:
Using the exported name in `src/shared/components/BackToTheTop.tsx`, then used within a querySelector, so having a `#` messed it up. It's only for dev so I used a character that would not be clashing with querySelector and would still be quite unique, so a bug, for debugging...